.github/PULL_REQUEST_TEMPLATE

```diff
@@ -16,14 +16,4 @@ Add a description of your changes here.
 - [ ] safecommit shows 0 violations
 - [ ] Unit, integration, acceptance tests as appropriate
 - [ ] Performance
-- [ ] Database migrations are backwards-compatible
-
-### Reviewers
-If you've been tagged for review, please check your corresponding box once you've given the :+1:.
-- [ ] Code review: @Reviewer1
-- [ ] Code review: @Reviewer2
-- [ ] UI strings/error msgs review: @...
-- [ ] UX review: @...
-- [ ] Accessibility review: @...
-- [ ] Product review: @...
-- [ ] FYI @someone
+- [ ] Database migrations are backwards-compatible
```

Comment:
Always leave a newline after the file, I hate to see that annoying mark: 

 - <img width="37" alt="screen shot 2016-12-19 at 6 32 01 pm" src="https://cloud.githubusercontent.com/assets/645156/21320368/75dffcea-c619-11e6-9c63-5a8a87b9a93b.png">